deployment/hasura/metadata/databases/AerieMerlin/tables/public_simulation_extent.yaml

```diff
@@ -0,0 +1,19 @@
+table:
+  name: simulation_extent
+  schema: public
+select_permissions:
```

Comment:
Would the reverse object relationship have any value (sim_extent -> sim_dataset)?

Reply:
I can't think of a use case for that relationship - let me know if you have one in mind